recipes/qwt/all/test_package/example.cpp

```diff
@@ -5,6 +5,6 @@
 
 int main()
 {
-    qDebug() << QwtDate::toString(QwtDate::toDateTime(10), "MMM dd hh:mm ", QwtDate::FirstThursday);
+    //qDebug() << QwtDate::toString(QwtDate::toDateTime(10), "MMM dd hh:mm ", QwtDate::FirstThursday);
```

Comment:
Could you please remind me why the test does not call any function now ? It is not a good thing, because it means the linker will simply discard qwt lib and potential missing dependencies won't be caught